OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -35,10 +35,10 @@ public class MissileInfo : IProjectileInfo
 		public readonly bool Shadow = false;
 
 		[Desc("Minimum vertical launch angle (pitch).")]
-		public readonly WAngle MinimumLaunchAngle = WAngle.Zero;
+		public readonly WAngle MinimumLaunchAngle = new WAngle(-64);
 
 		[Desc("Maximum vertical launch angle (pitch).")]
-		public readonly WAngle MaximumLaunchAngle = new WAngle(64);
+		public readonly WAngle MaximumLaunchAngle = new WAngle(128);
```

Comment:
This still seems to be insufficient for the Advanced Guard Tower to target Orcas smoothly. 128 corresponds to a 45° pitch w.r.t. the horizontal. To enable targeting directly upwards you need a 90°, ie, a value of 256